pandas/core/series.py

```diff
@@ -4565,6 +4565,13 @@ def to_period(self, freq=None, copy=True) -> "Series":
     index: "Index" = properties.AxisProperty(
         axis=0, doc="The index (axis labels) of the Series."
     )
+    _AXIS_ORDERS = ["index"]
+    _AXIS_NUMBERS = {"index": 0}
+    _AXIS_NAMES = {0: "index"}
+    _AXIS_REVERSED = False
+    _info_axis_number = 0
+    _info_axis_name = _AXIS_ORDERS[_info_axis_number]
```

Comment:
i _think_ should either be a property in NDFrame or just be explicit, i.e `_info_axis_name = "index"`